exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -37,10 +38,14 @@ var testData = []struct {
 		""},
 	{4, 10, Product{}, Product{}, "No palindromes"},
 	{10, 4, Product{}, Product{}, "fmin > fmax"},
-	/* bonus curiosities.  (can a negative number be a palindrome?
-	// most say no
-	{-99, -10, Product{}, Product{}, "Negative limits"},
-	// but you can still get non-negative products from negative factors.
+
+	/* Bonus curiosities.  Can a negative number be a palindrome? Most say no
+	{-99, -10,
+		Product{},
+		Product{},
+	"Negative limits"},
```

Comment:
oh no my mistake, sorry! I casted only the `^uint(0)` to `int(^uint(0))` so leaving out the bit shift and so I had the error.
Your solution works fine! I will update the PR :) Sorry for that.